gsoc/models.py

```diff
@@ -0,0 +1,222 @@
+    org_payment_details_can_be_changed = models.BooleanField()
+    admins_can_register = models.BooleanField()
+    mentor_first_evals_can_be_submitted = models.BooleanField()
+    students_can_edit_project = models.BooleanField()
+    tax_forms_can_be_uploaded = models.BooleanField()
+    accepted_projects_published = models.BooleanField()
+    student_first_evals_can_be_submitted = models.BooleanField()
+    second_enrollment_forms_can_be_submitted = models.BooleanField()
+    enrollments_can_be_reviewed = models.BooleanField()
+    mentor_second_evals_can_be_submitted = models.BooleanField()
+    slots_can_be_assigned = models.BooleanField()
+    second_eval_emails_can_be_sent = models.BooleanField()
+    orgs_can_be_edited = models.BooleanField()
+    mentor_finals_can_be_submitted = models.BooleanField()
+    slot_allocation_emails_can_be_sent = models.BooleanField()
+    projects_can_be_accepted = models.BooleanField()
+    enrollments_can_undergo_second_review = models.BooleanField()
+    orgs_can_be_accepted = models.BooleanField()
+    students_can_register = models.BooleanField()
+    second_enrollments_can_be_reviewed = models.BooleanField()
+    second_enrollment_forms_can_be_re_submitted = models.BooleanField()
+    org_members_can_edit_proposal = models.BooleanField()
+    evaluations_in_progress = models.BooleanField()
+    proposal_can_be_deleted = models.BooleanField()
+    accepted_orgs_published = models.BooleanField()
+    results_published = models.BooleanField()
+    send_assignee_emails = models.BooleanField()
+    org_members_can_view_final_proposals = models.BooleanField()
+    orgs_can_register = models.BooleanField()
+    student_second_evals_can_be_submitted = models.BooleanField()
+    enrollment_forms_can_be_submitted = models.BooleanField()
+    project_decisions_emails_can_be_sent = models.BooleanField()
+    orgs_can_see_enrollment_status = models.BooleanField()
+    proposal_in_progress = models.BooleanField()
+    orgs_have_been_accepted = models.BooleanField()
+    mentors_can_register = models.BooleanField()
+    final_evaluations_emails_can_be_sent = models.BooleanField()
+    slots_can_be_requested = models.BooleanField()
+    org_accept_reject_emails_can_be_sent = models.BooleanField()
+    org_payment_details_visible = models.BooleanField()
+    students_can_submit_proposals = models.BooleanField()
+    student_withdrawals_trigger_email = models.BooleanField()
+    enrollment_forms_can_be_re_submitted = models.BooleanField()
+    tax_forms_can_be_reviewed = models.BooleanField()
+    first_eval_emails_can_be_sent = models.BooleanField()
+    second_review_status_visible = models.BooleanField()
+    project_decisions_can_be_finalized = models.BooleanField()
+    student_home_address_is_editable = models.BooleanField()
+    result = models.ForeignKey(Result, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return str(self.result)
+
+
+class Copy(models.Model):
```

Comment:
Why is this model called Copy? 
PS: ignore me if this name has been discussed on gitter

Reply:
Makes sense to call it something like Statistics?

Reply:
This should be ProgramStatistics